azure-pipelines/build.yml

```diff
@@ -1,13 +1,18 @@
 name: $(Date:yyyyMMdd).$(Rev:r)
+parameters:
+- name: OfficialRelease
+  displayName: "Build and publish an official release"
+  type: boolean
+  default: false
 variables:
+- name: Build.OfficialRelease
+  value: ${{ parameters.OfficialRelease }}
 - name: Codeql.Enabled
   value: true
-- name: PublishNuGetOrg
-  value: false
 - name: TeamName
   value: vscx-tools-authoring
```

Comment:
Not sure what TeamName should be here. Authoring seems wrong though.

Reply:
@abpiskunov Any ideas? Perhaps just `upgrade-assistant`?

Reply:
looks like maybe `dotnetupgradeassistant`?

Reply:
Works for me